### ISSUE TITLE
Reversed params

### ISSUE THREAD

**Issue by naps62** [iden3/snarkjs#148]
Why are proof and pub being reversed in here?

https://github.com/iden3/snarkjs/blob/31f7867fa031c7e2c056331a7a3d4c6c29d5de35/build/main.cjs#L1264-L1266

**Comment by phated** [comment]
Looks like a bug from #146 - want to submit a fix?

**Comment by naps62** [comment]
@phated certainly. will do that now. I was swamped in a hackathon, now I have a bit more time :)
